apps/backend/src/schemas/configuration.py

```
@@ -0,0 +1,549 @@
+
+        # Validate restart services are subset of affected services
+        if self.restart_services and self.affected_services:
+            for service in self.restart_services:
+                if service not in self.affected_services:
+                    raise ValueError(
+                        f'Restart service "{service}" must be in affected_services list'
+                    )
+
+        # Validate confidence score consistency
+        if self.confidence_score is not None:
+            if self.confidence_score < 0.5 and self.processed:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)